sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -82,6 +103,109 @@ With this configuration, the Azure Monitor Distro will use the credentials of th
 
 Note that the `Credential` property is optional. If it is not set, Azure Monitor Distro will use the Instrumentation Key from the Connection String to send data to Azure Monitor.
 
+### Advanced configuration
+
+The Azure Monitor Distro includes .NET OpenTelemetry instrumentation for [ASP.NET Core](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNetCore/), [HttpClient](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Http/), and [SQLClient](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.SqlClient). However, you can customize the instrumentation included or use additional instrumentation on your own using the OpenTelemetry API. Here are some examples of how to customize the instrumentation:
+
+#### Customizing ASP.NET Core Instrumentation Options
+
+```C#
+builder.Services.AddOpenTelemetry().UseAzureMonitor();
+builder.Services.Configure<AspNetCoreInstrumentationOptions>(options =>
+{
+    options.RecordException = true;
+    options.Filter = (httpContext) =>
+    {
+        // only collect telemetry about HTTP GET requests
+        return httpContext.Request.Method.Equals("GET");
+    };
+});
+```
+
+#### Customizing HttpClient Instrumentation Options
+
+```C#
+builder.Services.AddOpenTelemetry().UseAzureMonitor();
+builder.Services.Configure<HttpClientInstrumentationOptions>(options =>
+{
+    options.RecordException = true;
+    options.FilterHttpRequestMessage = (httpRequestMessage) =>
+    {
+        // only collect telemetry about HTTP GET requests
+        return httpRequestMessage.Method.Equals("GET");
```

Comment:
Is this the idiomatic way? (or https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.httpmethods.isget?view=aspnetcore-7.0)

Reply:
Found a similar discussion https://github.com/IdentityServer/IdentityServer4/issues/2098